camelot/core/utils.py

```diff
@@ -79,18 +80,14 @@ def set_translation(source, value):
 def load_translations():
```

Comment:
Tim, aub nog volgende aanpassingen : 

- geef de session of de 'connectable' mee als argument van de functie, zodat er geen ambiguiteit is
- gebruik toch die 'select' ipv een 'query', zodat er geen 'object overhead' is, mogelijk zit er veel in de db, en die Translation objecten hoeven ook nt in de session te geraken